<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectSystemDialog</class>
 <widget class="QDialog" name="SelectSystemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select default image</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="dialogLabel">
     <property name="text">
      <string>Select one of the following CANNOLO images to set as default:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="isoList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectButton">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
